sass/_index.scss

```diff
@@ -1,335 +1,424 @@
-.hero {
-    color: #fff;
-    background: linear-gradient(-45deg, #EE7752, #E73C7E, #23A6D5, #23D5AB);
-    animation: Gradient 10s ease infinite;
-    background-size: 400% 400%;
-
-    display: flex;
-    height: 600px;
-    padding-top: 60px;
-    flex-direction: column;
-    justify-content: center;
-    flex: none;
-
-    .background-img {
-        position: absolute;
-        left: 0;
-        top: 60px;
-        right: 0;
-        bottom: 0;
-        height: 600px;
-        background-image: url('/assets/hero-spiro.svg');
-        background-position: 100% 50%;
-        background-size: cover;
-        background-repeat: repeat;
-        background-attachment: scroll;
+#frontpage {
 
+    #hero {
+        height: calc(100vh - 60px);
+        background-color: #000;
+        background-image: url("/assets/frontpage/hero_dancing_lines.svg");
         @media (max-width: 767px) {
-            background-position: 50% 50%;
+            background-image: url("/assets/frontpage/hero_dancing_lines_mobile.svg");
         }
+        background-size: 100% 100%;
+        background-repeat: no-repeat;
+        position: relative;
+        overflow: hidden;
+
+        .hero_content {
+            height: 100%;
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+
+            h2 {
+                color: #fff;
+                font-size: 40px;
```

Comment:
I would probably declare this relative to the parent, but not super important.

Reply:
I'm not sure which part would be relative to the parent :) ?
Do you mean the font-size, in `(r)em` instead of `px`?

Reply:
Yes, I mean having font-sizes (except at the root) in `%` or `em`.